tests/unit/preprocessor/_multimodel/test_multimodel.py

```diff
@@ -1,256 +1,331 @@
-"""Unit test for :func:`esmvalcore.preprocessor._multimodel`."""
+"""Unit test for :func:`esmvalcore.preprocessor._multimodel`"""
 
-import unittest
+from datetime import datetime
 
 import iris
 import numpy as np
+import pytest
 from cf_units import Unit
+from iris.cube import Cube
 
-import tests
+import esmvalcore.preprocessor._multimodel as mm
 from esmvalcore.preprocessor import multi_model_statistics
-from esmvalcore.preprocessor._multimodel import (
-    _assemble_data,
-    _compute_statistic,
-    _get_time_slice,
-    _multiproduct_statistics,
-    _plev_fix,
-    _put_in_cube,
-    _unify_time_coordinates,
+
+SPAN_OPTIONS = ('overlap', 'full')
+
+FREQUENCY_OPTIONS = ('daily', 'monthly', 'yearly')  # hourly
+
+CALENDAR_OPTIONS = ('360_day', '365_day', 'gregorian', 'proleptic_gregorian',
+                    'julian')
+
+
+def assert_array_almost_equal(this, other):
+    """Assert that array `this` almost equals array `other`."""
+    if np.ma.isMaskedArray(this) or np.ma.isMaskedArray(other):
+        np.testing.assert_array_equal(this.mask, other.mask)
+
+    np.testing.assert_allclose(this, other)
```

Comment:
any reason why you add the same function as the one from the tests' init file here? well, you are adding the allclose check to it, but why do you need it? please use the imported function and if there is a need for an almost equal check, add it in the test that needs it

Reply:
Hi @valeriupredoi , I'm not entirely sure how to import this file without messing with `sys.path`. The default mechanics for defining imports is in conftest.py, which we do not use atm.

Reply:
no need to import per se it's already imported at module run level, just use it from there without duplicating it here

Reply:
Like you said, the functions are different (`array_allclose` vs `array_equal`), so the other one cannot be used directly. Instead, I changed the function name to better reflect its function. Does that work for you?

Reply:
OK we should move this one upstream in the init at some point and just use it from there